test/FoldersApiIntegrationTests.js

```diff
@@ -0,0 +1,108 @@
+const docusign = require('../src/index');
```

Comment:
Until we're ready to drop support for node 4-5 we won't be able to move forward with keywords such as const and let